lib/ocpp/common/websocket/websocket_libwebsockets.cpp

```diff
@@ -779,38 +776,8 @@ bool WebsocketLibwebsockets::connect() {
         // sent over the wire on the client_loop, not giving the opportunity to the loop to
         // advance we will have a dead-lock
         this->recv_message_thread = std::make_unique<std::thread>(&WebsocketLibwebsockets::recv_loop, this);
-
-        // Wait until connect or timeout
-        timeouted = !conn_cv.wait_for(lock, std::chrono::seconds(60), [&]() {
-            return !local_data->is_connecting() && EConnectionState::INITIALIZE != local_data->get_state();
-        });
-
-        connected = (local_data->get_state() == EConnectionState::CONNECTED);
     }
-
-    if (!connected) {
-        EVLOG_info << "Connect failed with state: " << (int)local_data->get_state() << " Timeouted: " << timeouted;
-
-        // If we timeouted the on_conn_fail was not dispatched, since it did not had the chance
-        if (timeouted && local_data->get_state() != EConnectionState::ERROR) {
-            EVLOG_error << "Conn failed with timeout, without disconnect dispatch, dispatching manually.";
-            on_conn_fail();
```

Comment:
So this is not called anymore when there is a timeout (because there is no timeout anymore). Isn't that a problem?